.github/workflows/build-starter-templates.yml

```diff
@@ -62,3 +60,16 @@ jobs:
               --body "The **${{ matrix.template }}** starter template failed to build using **${{ matrix.package-manager }}**.\n\nCheck the full workflow logs for more details: [View Workflow Run]($RUN_URL)" \
               --labels "bug"
           fi
+
+  check-failures:
```

Comment:
not sure what value this job is adding... is this because of the "continue on error" setting? if yes, add a comment to explain it